etc/tendrl/node-agent/node-agent.conf.yaml.sample

```diff
@@ -5,4 +5,4 @@ tendrl_ansible_exec_file: $HOME/.tendrl/node-agent/ansible_exec
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
 log_level: DEBUG
 package_source_type: rpm
-logging_socket_path: "/var/run/tendrl/.message.sock."
+logging_socket_path: /var/run/.tendrl.message.sock
```

Comment:
@nthomas-redhat i got your point i will update this patch as your per you suggestion